source/extensions/dependence.c

```diff
@@ -252,9 +260,13 @@ char* osl_dependence_sprint(osl_dependence_p dependence) {
    ;
   snprintf(buff, OSL_MAX_STRING, "# Number of dependences\n%d\n", nb_deps);
   strcat(buffer, buff);
-  
+ 
+  if ((n < 0) || (n > nb_deps)) {
```

Comment:
Explain this behavior in the documentation, especially negative one since it may be ambiguous.  (Someone may expect negatives to print "all but the last" python-style)